tests/utils/getClaimStatus.test.tsx

```diff
@@ -1,14 +1,32 @@
-import { getClaimStatusDescription } from '../../utils/getClaimStatus'
+import { getClaimStatusHeading, getClaimStatusSummary } from '../../utils/getClaimStatus'
 import { ScenarioType } from '../../utils/getScenarioContent'
 import { getNumericEnumKeys } from '../../utils/numericEnum'
 
-// Test getClaimStatusDescripton()
-describe('Getting the Claim Status description', () => {
-  it('returns the correct description for the scenario', () => {
+// Test getClaimStatusHeading()
+describe('The Claim Status heading', () => {
+  it('is correct for each scenario', () => {
     for (const key of getNumericEnumKeys(ScenarioType)) {
-      expect(getClaimStatusDescription(key)).toEqual(
-        expect.stringMatching(/claim-status:scenarios.scenario[0-9]+.description/),
-      )
+      expect(getClaimStatusHeading(key)).toEqual(expect.stringMatching(/claim-status:scenarios.scenario[0-9]+.heading/))
     }
   })
 })
+
+// Test getClaimStatusSummary()
+// Note: These tests test against real content
```

Comment:
@lomky I'm feeling a little 😬 about these tests that require real content (i.e. `public/locales/en/claim-status.json`, but not sure what the better solution is since the code explicitly relies on that file. I did write this [alternative set of tests](https://github.com/cagov/ui-claim-tracker/pull/314/commits/de46afe6951b358681783ad32a7ccecf480fa8c4). Let me know if you think this is better (and then help me resolve the typescript errors/warnings in them...)

Reply:
> is the problem here that we're ultimately testing the middle part of what's ultimately a visual assertion? Like what do these tests give us that the snapshot tests won't also catch?

🤔 Good point. Since we're doing snapshot tests for the `<ClaimStatus>` component for all the scenarios, I think these tests aren't actually that useful.